trading_calendars/exchange_calendar_xnze.py

```diff
@@ -114,6 +114,20 @@
     start_date="2011-01-01",
 )
 
+# Jevons Global: further ad-hoc holidays and closures
```

Comment:
It looks like `Timestamp` isn't yet imported in this file, could you add that?

Reply:
@jmccorriston sorry for the delay... I just have a rebuild to do

Reply:
@jmccorriston added import for Timestamp to trading_calendars/exchange_calendar_xnze.py